pkgs/development/libraries/LAStools/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake }:
+stdenv.mkDerivation rec {
+
+  version = "201003";
+  name = "LAStools-${version}";
+
+  src = fetchFromGitHub {
+    owner = "LAStools";
+    repo = "LAStools";
+    rev = "635b76b42cc4912762da31b92f875df5310e1714";
```

Comment:
Are there tags upstream?

Reply:
No.

![Screenshot from 2020-11-10 10-24-36](https://user-images.githubusercontent.com/2990492/98715530-25432000-233f-11eb-9556-f795c9e84120.png)

But the project seems to communicate releases by date: https://lastools.github.io/download/CHANGES.txt